packages/frontend-api/src/Resources/config/graphql-types/Pagination/EdgeInterface.types.yml

```diff
@@ -0,0 +1,8 @@
+EdgeInterface:
```

Comment:
Did you consider connection-relay? https://github.com/overblog/GraphQLBundle/blob/master/docs/definitions/relay/connection.md

Reply:
I changed it :slightly_smiling_face: